docs/resources/virtual_server.md

```diff
@@ -1,23 +1,37 @@
 ---
-page_title: "virtual_server resource - terraform-provider-anxcloud"
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud_virtual_server Resource - terraform-provider-anxcloud"
 subcategory: ""
 description: |-
-  The Virual Server resource allows you to create virtual machines at Anexia Cloud.
+  The virtual_server resource allows you to configure and run Virtual Machine at Anexia Cloud.
+  Known limitations
+  removal of disks not supportedremoval of networks not supported
```

Comment:
can we remove those from the header and only have them in text below?

Reply:
I've removed it manually.